src/python/grpcio/grpc/_cython/_cygrpc/channel.pyx.pxi

```diff
@@ -142,10 +144,10 @@ cdef _cancel(
       _check_and_raise_call_error_no_metadata(c_call_error)
 
 
-cdef BatchOperationEvent _next_call_event(
+cdef _next_call_event(
```

Comment:
Which other type this function can return in addition to BatchOperationEvent?

Reply:
`_latent_event` returns the result of `_interpret_event`. With the optional deadline, the returned event type can be a `ConnectivityEvent` in the event of a timeout.

Reply:
thanks